training/README.rst

```diff
@@ -0,0 +1,163 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+========
+Training
+========
+
+This module was written to extend the functionality of events to support
+trainings and allow you to manage training actions, training groups,
```

Comment:
Training should not be used in plural (https://en.wiktionary.org/wiki/training).
=> multiple training sessions